src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c

```diff
@@ -137,7 +137,7 @@ u8 WeatherTag_CheckEnableWeatherEffect(EnWeatherTag* this, GlobalContext* global
 
     if (Actor_WorldDistXZToActor(&player->actor, &this->actor) < WEATHER_TAG_RANGE100(this->actor.params)) {
         if ((globalCtx->envCtx.unk_1E != 0) || !D_8011FB3C ||
-            (globalCtx->skyboxId != 1 && globalCtx->envCtx.unk_1F == globalCtx->envCtx.unk_20)) {
+            (globalCtx->skyboxId != SKYBOX_CLEAR_SKY && globalCtx->envCtx.unk_1F == globalCtx->envCtx.unk_20)) {
```

Comment:
I'm slightly iffy on this name because if the weather is set to cloudy than the sky is not clear